nature/forms.py

```diff
@@ -12,8 +12,18 @@ class FeatureForm(forms.ModelForm):
     feature_class = forms.ModelChoiceField(
         queryset=FeatureClass.objects.order_by('name'),
         label=FeatureClass._meta.verbose_name.capitalize(),
+        help_text='Mihin kohdeluokkaan kohde kuuluu',
     )
 
+    def __init__(self, *args, **kwargs):
+        super(FeatureForm, self).__init__(*args, **kwargs)
+        if not self.initial:
+            self.fields['geometry'].help_text = """
```

Comment:
I wonder why we do not specify the help text for geometry field in meta options as other fields?

Reply:
As the help text in this case only applies to the controls, which are not present when one is editing an existing feature (as far as I've seen at least)

Reply:
Not blocking the merge. Just realized that the help text concerns about when creating features, but I think it would be good to add texts for editing existing feature by adding text like "Use can change the type geometries by clicking `Remove all geomtries` below the map and then select new type of geometries on top right corner".

Reply:
The text is now always shown and has an extra text to help in the case the user is editing :)